public/UI.js

```diff
@@ -184,4 +185,22 @@ class UIController {
             .querySelector(DOMstrings.codingProgressContainer)
             .insertAdjacentHTML("beforeend", newHtml);
     }
+
+    static addSystemsGraphs() {
+        let DOMstrings = UIController.getDOMstrings(),
+            html = `<div class="container"> 
```

Comment:
Can you add last updated timestamp (and red warning) just like we have for the traffic graphs?

Reply:
I've added